docs/root/configuration/access_log.rst

```diff
@@ -164,6 +164,13 @@ The following command operators are supported:
   TCP
     Not implemented ("-").
 
+%TRAIL(X?Y):Z%
```

Comment:
Thoughts on just using `%TRAILER` here? It's 2 more chars and I think more clear (and more foreign than REQ and RESP).

Reply:
Renamed to TRAILER. Abbreviations are very contagious :)

Reply:
Renamed to TRAILERS. abbreviations are so contagious.